frontend/components/GroupedProducts/components/Product/index.jsx

```diff
@@ -80,6 +86,7 @@ const Product = ({
                     noShadow: renderFlatButtons,
                     type: ADD_TO_CART_BUTTON_TYPE_GROUPED,
                   }}
+                  conditioner={conditioner}
```

Comment:
Please add conditioner to the test of this component so it doesn't render warning.

Reply:
Ah right. Updated.